hosting_docs/source/reference/1-commcare-cloud/1-installation.rst

```diff
@@ -117,12 +117,12 @@ If you get to this point, congratulations! ``commcare-cloud`` is installed.
 Manual Installation
 -------------------
 
-You will need python 3.6 installed to follow these instructions (for
-Ubuntu 18.04; other operating systems may differ):
+You will need python 3.10 installed to follow these instructions (for
+Ubuntu 22.04; other operating systems may differ):
 
 .. code-block::
 
-   sudo apt-get install git python3-dev python3-pip
+   sudo apt-get install git python3.10-dev python3-pip
```

Comment:
I'm wondering if this section should point to the [instructions] for installing 3.10 since that ensures _everything_ that is necessary is installed. I don't like where that section on installing 3.10 currently lives though, so maybe this could be the impetus to move to a more logical spot that both sets of directions can reference?

(a) I think we need to be explicit about python3.10 because this is outside of a virtual environment, and we don't know for sure what version python3 references. 
b) We are shifting to removing the dependency on `virtualenv` since python has `venv` built-in. 
```suggestion
   python3.10 -m venv ~/.virtualenvs/cchq
```) for installing 3.10 given that the current config that we support is 3.10 on 18.04.

Reply:
> this section should point to the [instructions] for installing 3.10

I like that idea, can you point me to those instructions?

Reply:
🤦🏻 updated my initial comment to actually have the link, but here are [the instructions](https://commcare-cloud.readthedocs.io/en/latest/installation/2-manual-install.html#upgrade-to-python-3-10).

Reply:
I think it's [here](https://github.com/dimagi/commcare-cloud/blob/master/hosting_docs/source/installation/2-manual-install.rst#upgrade-to-python-310). As an example, I think `python3.10-venv` (apt package) is needed for `python3.10 -m venv ...`

Reply:
See latest changes.